packages/simple-hub/package.json

```diff
@@ -66,6 +65,6 @@
     "prettier:check": "npx prettier --check 'src/**/*.{ts,tsx}'",
     "prettier:write": "npx prettier --write 'src/**/*.{ts,tsx}'",
     "start:shared-ganache": "NODE_ENV=development npx start-shared-ganache",
-    "test:ci": "run-s chain:test hub:test"
+    "test:ci": "yarn hub:test"
```

Comment:
Should we still be skipping `SimpleHub` tests?

Reply:
`chain:test` is testing facilities that we are no longer using -- watching for deposits, and matching them.